spec.html

```diff
@@ -24918,7 +24924,7 @@ <h1>ECMAScript Standard Built-in Objects</h1>
     <p>For example, the function object that is the initial value of the *"map"* property of the Array prototype object is described under the subclause heading &laquo;Array.prototype.map (callbackFn [ , thisArg])&raquo; which shows the two named arguments callbackFn and thisArg, the latter being optional; therefore the value of the *"length"* property of that function object is 1.</p>
   </emu-note>
   <p>Unless otherwise specified, the *"length"* property of a built-in function object has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.</p>
-  <p>Every built-in function object, including constructors, has a *"name"* property whose value is a String. Unless otherwise specified, this value is the name that is given to the function in this specification. Functions that are identified as anonymous functions use the empty String as the value of the *"name"* property. For functions that are specified as properties of objects, the name value is the property name string used to access the function. Functions that are specified as get or set accessor functions of built-in properties have *"get "* or *"set "* prepended to the property name string. The value of the *"name"* property is explicitly specified for each built-in functions whose property key is a Symbol value.</p>
+  <p>Every built-in function object, including constructors, has a *"name"* property whose value is a String. Unless otherwise specified, this value is the name that is given to the function in this specification. Functions that are identified as anonymous functions use the empty String as the value of the *"name"* property. For functions that are specified as properties of objects, the name value is the property name string used to access the function. Functions that are specified as get or set accessor functions of built-in properties have *"get "* or *"set "* prepended to the property name string. The value of the *"name"* property is explicitly specified for each built-in functions whose property key is a Symbol value. The *"name"* property is set using SetFunctionName.</p>
```

Comment:
What actually calls SetFunctionName for the built-in functions? Does 8.2.2 need to be amended?

Reply:
Discussed this on the editor call. Clause 17 should be amended to actually set [[InitialName]]. Clause 8.2.2 does not need to change, but could use clarification that it is _not_ steps, but is a description of the how the values in the table were created.

Reply:
What kind of change are you looking for here? I already amended it to explicitly say what logic should be used to set the name.

Reply:
I guess with https://github.com/tc39/ecma262/pull/2116 we don't need to explicitly say how to set the name here because we already say they should be created with CreateBuiltinFunction?